packages/transformers/webextension/src/WebExtensionTransformer.js

```diff
@@ -216,6 +216,46 @@ async function collectDependencies(
     }
     program.web_accessible_resources = war;
   }
+  if (program.declarative_net_request?.rule_resources) {
```

Comment:
If I'm right that `rule_resources` is a required field you just need to check for `program.declarative_net_request` here.